data/presets/fields/generator/output/electricity.json

```diff
@@ -1,6 +1,6 @@
 {
     "key": "generator:output:electricity",
-    "type": "text",
+    "type": "typeCombo",
```

Comment:
I feel like this field might have been `text` to allow capital letters like "MW" in the value units. 
Can you test to make sure that taginfo will still return results with capital letters?  

If taginfo is preferring lowercase values, you might need to add this field to the uppercase regex here: https://github.com/openstreetmap/iD/blob/master/modules/services/taginfo.js#L263-L267

Reply:
It looks like `:output` is already in the regex, so there's no problem with capitalization.

However, I noticed that the field replaces spaces with underscores, which is undesirable for this key. I'll remove the type change from this PR and make a new issue.